docs/signals/configuration/index.md

```diff
@@ -32,7 +32,7 @@ To connect to Signals, you will need 4 values:
 
 | Value           | Description                             | Where to get it                 | Format                            |
 | --------------- | --------------------------------------- | ------------------------------- | --------------------------------- |
-| Signals API URL | The API URL for your Signals deployment | Provided by Snowplow            | `https://{{UUID}}.svc.snplow.net` |
+| Signals API URL | The API URL for your Signals deployment | Provided by Snowplow            | `https://{{UUID_SUBSTR}}.signals.snowplowanalytics.com` |
```

Comment:
Do we need to balance the last column, so that its all aligned on the longest column ?  Markdown is happy as is, but common style seems to be to make them visually pretty in the code as well.

Reply:
Updated the rest of the rows to align with the updated column